docs/_static/custom.js

```diff
@@ -19,10 +19,24 @@ function openModal(modal) {
   modal.style.removeProperty('display');
 }
 
+function updateSetting(element) {
+  localStorage.setItem(element.name, element.checked);
+}
+
+function setFont(useSans) {
+  if (useSans === true) {
+    document.body.classList.add('sans');
+  } else {
+    document.body.classList.remove('sans');
+  }
+}
+
+const settings = [['useSansFont', setFont]];
```

Comment:
Couldn't this be an object? You can iterate through an object's keys and values with `Object.entries`.